marketplace/deployer_helm_tiller_base/bin/deploy_internal.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eox pipefail
+
+app_uid=$(kubectl get "applications/$NAME" \
+  --namespace="$NAMESPACE" \
+  --output=jsonpath='{.metadata.uid}')
+app_api_version=$(kubectl get "applications/$NAME" \
+  --namespace="$NAMESPACE" \
+  --output=jsonpath='{.apiVersion}')
+
+for chart in /data/chart/*; do
+  # Expand and apply the template for the Application resource.
+  # Note: This must be done out-of-band because the Application resource
+  # is created before the deployer is invoked, but helm does not handle
+  # pre-existing resources.
+  helm template \
+      --name="$NAME" \
+      --namespace="$NAMESPACE" \
+      --values=<(print_config.py --output=yaml) \
+			--set "$(cat /data/schema.yaml \
+                 | yaml2json \
+                 | jq -r '.properties
```

Comment:
Let's limit the use of `jq`, hard to read to most and not testable. Can you update `print_config.py` to add support for a new flag `--key`. It should be similar to `--param`, except it prints the key instead of the value.

Reply:
I'd prefer to do this as a follow up - it's not partner facing.

Reply:
Tracked in https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/issues/237.